### ISSUE TITLE
FEAT: Browse custom data types in PostgreSQL

### ISSUE THREAD

**Issue by avaly** [beekeeper-studio/beekeeper-studio#805]
**Is your feature request related to a problem? Please describe.**

It would be nice to at least view the existing data types defined in a PostgreSQL database (e.g. [`ENUM` types](https://www.postgresql.org/docs/current/datatype-enum.html).

**Describe the solution you'd like**

List the existing custom data types in the sidebar below the tables and allow to view their definition in the main screen.

**Comment by rathboma** [comment]
Yes I agree. It's not an immediate priority (unless we get a PR!), but it's on our radar.

**Comment by avaly** [comment]
I'm happy to contribute a PR.

**Comment by busseyl** [comment]
Is this still a feature under consideration? It would be a great addition to an already amazing app!

**Comment by ahopkins** [comment]
Would love to see this. Seeing how old the issue is :disappointed:
